pkg/bufferdecoder/net_decoder.go

```diff
@@ -33,8 +33,8 @@ func (decoder *EbpfDecoder) DecodeNetEventMetadata(eventMetaData *NetEventMetada
 }
 
 type NetCaptureData struct {
-	PacketLength   uint32 `json:"pkt_len"`
-	InterfaceIndex uint32 `json:"if_index"`
+	PacketLength            uint32 `json:"pkt_len"`
+	CaptureConfigIfaceIndex uint32 `json:"if_index"`
```

Comment:
The "Capture" part is not necessary in my opinion, because it is part of the struct name